### ISSUE TITLE
Exception for Entities Card footer

### ISSUE THREAD

**Issue by lezmaka** [zanna-37/hass-swipe-navigation#41]
On the mobile Home Assistant app (specifically iOS) when the footer of an entities card is set to contain buttons, rather than have multiple rows like on a desktop, it overflows into a horizontal scrolling list.  I would imagine there would be a similar issue if the header was setup as buttons.

![IMAGE-1](https://user-images.githubusercontent.com/5054745/221601373-66e25007-eb4c-4b49-9054-ee37c8fb0994.jpg)

Let me know if you need any more info.

**Comment by zanna-37** [comment]
Hi, @lezmaka, thank you for spotting that! Can you share the complete yaml of the card?

**Comment by lezmaka** [comment]
The one in the screenshot is somewhat complex (to me anyway).  Here's a simpler card that has the same issue.  Yaml for the card in screenshot will be at the bottom if you need it.

Simpler card for bedroom
```
type: entities
entities:
  - type: custom:slider-entity-row
    entity: light.bd11
    tap_action:
      action: toggle
    hold_action:
      action: more-info
  - type: custom:slider-entity-row
    entity: light.bd22
    tap_action:
      action: toggle
    hold_action:
      action: more-info
  - type: custom:slider-entity-row
    entity: light.bd_3
    tap_action:
      action: toggle
    hold_action:
      action: more-info
state_color: true
show_header_toggle: true
title: Bedroom
footer:
  type: buttons
  entities:
    - entity: scene.bedroom_sunny
      name: Sunny
      icon: hue:scene-bright
      tap_action:
        action: toggle
    - entity: scene.bedroom_bright
      name: Bright
      icon: hue:scene-bright
      tap_action:
        action: toggle
    - entity: scene.bedroom_relax
      name: Relax
      icon: hue:scene-relax
      tap_action:
        action: toggle
    - entity: scene.bedroom_rest
      name: Rest
      icon: hue:scene-dimmed
      tap_action:
        action: toggle
    - entity: scene.bedroom_bedroom_torch
      name: Torch
      icon: hue:scene-dimmed
      tap_action:
        action: toggle
```



Yaml of card in screenshot
```
type: entities
entities:
  - type: custom:slider-entity-row
    entity: light.lamp_1
    icon: hue:bulb-classic
    hold_action:
      action: more-info
  - type: custom:slider-entity-row
    entity: light.strip_1
    icon: hue:lightstrip
    hold_action:
      action: more-info
  - type: custom:slider-entity-row
    entity: light.strip_2
    icon: hue:lightstrip
    hold_action:
      action: more-info
  - type: custom:slider-entity-row
    entity: light.backlight
    icon: hue:lightstrip
    hold_action:
      action: more-info
  - type: custom:slider-entity-row
    entity: light.hue_play_4
    icon: hue:play-bar
    hold_action:
      action: more-info
  - type: section
    label: Adaptive Lighting
  - entity: switch.adaptive_lighting_living_room_adaptive
  - type: conditional
    conditions:
      - entity: switch.adaptive_lighting_living_room_adaptive
        state: 'on'
    row:
      type: attribute
      entity: switch.adaptive_lighting_living_room_adaptive
      attribute: manual_control
      name: Excluded
      icon: none
  - entity: automation.living_room_adaptive_auto_reset
    name: Auto Reset
    state_color: false
  - entity: input_boolean.living_room_adaptive_reset_enabled
    name: Time Reset
    state_color: false
  - entity: input_datetime.living_room_adaptive_reset_time
    name: Reset Time
  - entity: script.living_room_adaptive_reset
    name: Reset
footer:
  type: buttons
  entities:
    - entity: scene.living_room_bright
      name: Bright
      icon: hue:scene-bright
      tap_action:
        action: toggle
    - entity: scene.living_room_relax
      name: Relax
      icon: hue:scene-relax
      tap_action:
        action: toggle
    - entity: scene.living_room_rest
      name: Rest
      icon: mdi:sleep
      tap_action:
        action: toggle
    - entity: scene.living_room_top_relax
      name: Top Relax
      tap_action:
        action: toggle
    - entity: scene.living_room_top_night_1
      name: Top Night 1
      icon: mdi:brightness-2
      tap_action:
        action: toggle
    - entity: scene.living_room_top_night_2
      name: Top Night 2
      icon: mdi:brightness-3
      tap_action:
        action: toggle
    - entity: scene.living_room_concentrate
      name: Concentrate
      icon: hue:scene-concentrate
      tap_action:
        action: toggle
    - entity: scene.living_room_energize
      name: Energize
      icon: hue:scene-energize
      tap_action:
        action: toggle
state_color: true
show_header_toggle: true
title: Living Room

```

**Comment by zanna-37** [comment]
Everything should be fixed in the new pre-release version `v1.11.0`.
Let me know! =)

**Comment by lezmaka** [comment]
That worked, thanks!
